packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardRenderer.js

```diff
@@ -434,6 +450,9 @@ const AdaptiveCardRenderer = ({ actionPerformedClassName, adaptiveCard, disabled
   }, [adaptiveCard]);
 
   useEffect(() => {
+    setAttribute(adaptiveCardElementRef.current, 'aria-pressed', 'true') &&
```

Comment:
Are we intentionally ignoring the undo function?

Reply:
fixed